esphome/components/scd30/scd30.h

```diff
@@ -41,6 +43,7 @@ class SCD30Component : public PollingComponent, public i2c::I2CDevice {
   uint16_t altitude_compensation_{0xFFFF};
   uint16_t ambient_pressure_compensation_{0x0000};
   float temperature_offset_{0.0};
+  uint16_t update_interval_{0xFFFF};
```

Comment:
Can also remove the default `{0xFFFF}` from the header as set_update_interval will always be called

```suggestion
  uint16_t update_interval_;
```

Reply:
I prefer to have the config validation correctly as it was designed.